servicecontrol/installation.md

```diff
@@ -23,24 +23,32 @@ NOTE: ServiceControl should be installed on a separate dedicated machine with de
 
 ## Transport support
 
-In ServiceControl version 1.7 and above, the transport DLLs are managed by the installation and do not need to be downloaded from NuGet. ServiceControl can be configured to use one of the supported [transports](/transports/) listed below using the ServiceControl Management application:
+In ServiceControl version 1.7 and above, the transport packages are managed by the installation and do not need to be downloaded from NuGet. ServiceControl can be configured to use one of the supported [transports](/transports/) listed below using the ServiceControl Management application:
 
- * [Microsoft Message Queuing (MSMQ)](/transports/msmq/)
- * [Azure Storage Queues](/transports/azure-storage-queues/)
- * [Azure Service Bus](/transports/azure-service-bus/)
- * [Azure Service Bus .NET Standard](/transports/azure-service-bus-netstandard)
- * [SQL Server](/transports/sql/)
- * [RabbitMQ](/transports/rabbitmq/)
- * [Amazon SQS](/transports/sqs/)
+* [Microsoft Message Queuing (MSMQ)](/transports/msmq/)
+* [Azure Service Bus](/transports/azure-service-bus-netstandard)
+* [Azure Storage Queues](/transports/azure-storage-queues/)
+* [SQL Server](/transports/sql/)
+* [RabbitMQ](/transports/rabbitmq/)
+* [Amazon Simple Queue Service (SQS)](/transports/sqs/)
 
- Certain features of the transports are not supported natively by ServiceControl and will require a [transport adapter](/servicecontrol/transport-adapter).
- 
-For preliminary Azure Service Bus .NET Standard support, select the Azure Service Bus forwarding topology.
+### Legacy options
 
-Adding third party transports via the Management Utility is not supported. If MSMQ is the selected transport then ensure the service has been installed and configured as outlined in [Installing The Platform Components Manually](/platform/installer/offline.md#platform-installer-components-nservicebus-prerequisites).
+* [Azure Service Bus - Endpoint-oriented topology](/transports/azure-service-bus) - No migration option is currently available, contact support@particular.net for further guidance.
```

Comment:
I have same concern as for the announcement we're going to post (https://github.com/Particular/Announcements/pull/301/files#r227855567).

Reply:
See my comment on the other PR